includes/admin/helpers/tables/class-fees-table.php

```diff
@@ -72,24 +72,21 @@ public function prepare_items() {
         $this->_column_headers = array( $this->get_columns(), array(), $this->get_sortable_columns() );
 
         $args = array(
-			'admin_view'   => true, // Admin view shows all listings
+			'admin_view'   => true,
 			'enabled'      => 'all',
 			'include_free' => true,
         );
-
         $this->items = wpbdp_get_fee_plans( $args );
     }
 
     /** Rows **/
     public function single_row( $item ) {
-        $free_mode = ( ! wpbdp_payments_possible() );
         $classes   = '';
-
-        if ( $free_mode && $item->amount > 0.0 ) {
-            $classes .= 'disabled-fee';
-        } elseif ( 'free' === $item->tag ) {
-            $classes .= 'free-fee';
-        }
+		if ( $item->is_enabled_premium() ) {
+			$classes .= 'disabled-fee';
+		} elseif ( 'free' === $item->tag ) {
+			$classes .= 'free-fee';
```

Comment:
I don't think we need this class. Maybe you can look more into the js that uses this?
$('.wpbdp-admin-page-fees .wp-list-table.fees tbody').sortable({

Any idea when this table stopped being sortable?

Reply:
Seems the drag handle is missing, will try locate it

Reply:
The column `order` was missing. Checked all history and it wasn't there. Added it